scripts/upload-resources-to-github.sh

```diff
@@ -6,7 +6,6 @@ set -euo pipefail
 SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
 VERSION=$(make -s -f $SCRIPTPATH/../Makefile version)
 BUILD_DIR=$SCRIPTPATH/../build/cni-rel-yamls/$VERSION
-BINARY_DIR=$SCRIPTPATH/../build/bin
```

Comment:
Why this is not required now?

Reply:
I grepped and this was not used anywhere. I think we were at one time considering publishing binary artifacts (at this time we only publish compressed helm charts), but never ended up doing so.